vnext/Test/HttpServer.cpp

```diff
@@ -1,7 +1,8 @@
+// clang-format off
```

Comment:
Same comments as above related to clang-format. Would be best to do in a separate change, but if we want to change style, it might be nicer to just add a more specific clang-format file to whatever directory we want.

Reply:
Because these types are being refactored, I needed more control hence disabling clang-format made sense.

I wasn't aware the format files could be "overridden locally", but certainly will in a subsequent change.

Reply:
Julio, please let Yichen know about this format configuration, his pull requests also switch formatting around. We should have consistent formatting in our files, and from what I remember clang couldn't format lambda related code and macros correctly. Maybe it's better with new clang versions.

---
In reply to: [342892356](https://github.com/microsoft/react-native-windows/pull/3602#discussion_r342892356) [](ancestors = 342892356)